Formula/f/fltk@1.3.rb

```diff
@@ -0,0 +1,57 @@
+class FltkAT13 < Formula
+  desc "Cross-platform C++ GUI toolkit"
+  homepage "https://www.fltk.org/"
+  url "https://www.fltk.org/pub/fltk/1.3.10/fltk-1.3.10-source.tar.gz"
+  sha256 "c1c96d4f2ca7844f4b7945b4670aff2846f150cd5f3e23e3e4c70a61807108c7"
+  license "LGPL-2.0-only" => { with: "FLTK-exception" }
+
+  livecheck do
+    url "https://www.fltk.org/software.php"
+    regex(/href=.*?fltk[._-]v?(1\.3(?:\.\d+)*(?:-\d+)?)-source\.t/i)
+  end
+
+  keg_only :versioned_formula
+
+  depends_on "jpeg-turbo"
+  depends_on "libpng"
+
+  on_linux do
+    depends_on "pkgconf" => :build
+    depends_on "fontconfig"
+    depends_on "libx11"
+    depends_on "libxext"
+    depends_on "libxfixes"
+    depends_on "libxft"
+    depends_on "libxrender"
+    depends_on "libxt"
+    depends_on "mesa"
+    depends_on "mesa-glu"
+  end
+
+  def install
+    system "./configure", "--prefix=#{prefix}",
+                          "--enable-threads",
+                          "--enable-shared"
```

Comment:
Forgot to update this. Will do in a follow-up PR.

```suggestion
    args = %w[
      --enable-threads
      --enable-shared
    ]
    system "./configure", *args, *std_configure_args
```